### DISCUSSION TITLE
update using a composite primary key

### DISCUSSION THREAD

**Discussion by stausmar** [supabase/supabase#6536] [Questions] [answered]
Hi,

I use the following to update a specific record which is identified by the "id".

`const { data, error } = await supabase.from("products")
        .update({
            name, 
            description
        })
        .eq("id", params.id)
`

Let's assume my table "products" has not a single primary key, but the combination of two columns is the primary key. How would my update function look like? How can I identify this specific data record?

Thanks!

**Comment by GaryAustin1** [comment, marked as answer]
Just add another .eq or use .match or any combination of filters you desire.
